runners/core-java/src/test/java/org/apache/beam/runners/core/UnboundedReadFromBoundedSourceTest.java

```diff
@@ -292,7 +292,7 @@ private static void writeFile(File file, byte[] input) throws IOException {
    */
   private static class UnsplittableSource extends FileBasedSource<Byte> {
     public UnsplittableSource(String fileOrPatternSpec, long minBundleSize) {
-      super(fileOrPatternSpec, minBundleSize);
+      super(fileOrPatternSpec, /* TODO */ true, minBundleSize);
```

Comment:
Used for testing behavior around unsplittable sources. I think we can just pass in `true` here.